src/silx/math/_colormap.pyx

```diff
@@ -34,6 +34,7 @@ cimport cython
 from cython.parallel import prange
 cimport numpy as cnumpy
 from libc.math cimport frexp, sinh, sqrt
+from libc.math cimport pow as c_pow
```

Comment:
Using the `pow` function directly avoids issues and extra code that handles complex.